pages/common/kops.md

```diff
@@ -0,0 +1,24 @@
+# kops
+
+> A tool that create, destroy, upgrade and maintain production-grade, highly available, Kubernetes clusters from the command line.
```

Comment:
```suggestion
> A tool to create, destroy, upgrade and maintain Kubernetes clusters from the command line.
```
Do we really need all those adjectives? This sounds way simpler and more concise to me.